sqlglot/dialects/duckdb.py

```diff
@@ -1840,6 +1864,8 @@ def round_sql(self, expression: exp.Round) -> str:
                 if isinstance(decimals, exp.Literal):
                     if not decimals.is_int:
                         decimals = exp.cast(decimals, exp.DataType.Type.INT)
+                elif decimals.is_int:
+                    pass
```

Comment:
Should we add this term to the following `elif` branch instead? so `elif not is_int or is_type(...)`.